config/config-mp/src/main/java/io/helidon/config/mp/MpPropertiesMetaConfigProvider.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.config.mp;
+
+import java.util.List;
+import java.util.Set;
+
+import io.helidon.config.Config;
+import io.helidon.config.mp.spi.MpMetaConfigProvider;
+
+import org.eclipse.microprofile.config.spi.ConfigSource;
+
+/**
+ * Helidon MicroProfile meta-config provider for Properties configuration files.
+ */
+public class MpPropertiesMetaConfigProvider implements MpMetaConfigProvider {
```

Comment:
This class should not be public, as it is only used from this package and is not a "real" service provider implementation

Reply:
Removed public keyword